apps/api/v2/src/ee/bookings/2024-08-13/controllers/bookings.controller.ts

```diff
@@ -307,6 +308,28 @@ export class BookingsController_2024_08_13 {
     };
   }
 
+  @Get("/:bookingUid/getCalVideoSessions")
```

Comment:
1. If it is already a GET request then I think we can omit `get` from `getCalVideoSessions`.
2. We use kebab-case for endpoints so then the resulting request url should be
```
@Get("/:bookingUid/cal-video-sessions")
```
3. However, what if in the future we provide retrieving video sessions other than cal? So maybe we can have:
```
@Get("/:bookingUid/video-sessions")
```
which then would still use `getCalVideoSessions` because we don't support any other type and then in docs describe that this endpoint right now only can retrieve video sessions if the event type uses cal video.